### PR TITLE
Accessibility - Add placeholder/title for text areas

### PR DESCRIPTION
**Pull request by slafleche**
The textareas in Vanilla don't have a label. I've added a title and placeholder to help screen readers to announce what the text area is for for new posts. 

Closes: https://github.com/vanilla/vanilla/issues/6380

### CODE REVIEW

File: applications/vanilla/views/post/discussion.php [hunk 1]
```diff
@@ -44,7 +44,7 @@
     $this->fireEvent('BeforeBodyInput');
 
     echo '<div class="P">';
-    echo $this->Form->bodyBox('Body', ['Table' => 'Discussion', 'FileUpload' => true]);
+    echo $this->Form->bodyBox('Body', ['Table' => 'Discussion', 'FileUpload' => true, 'placeholder' => t('Type your message'), 'title' => t('Type your message')]);
```
Comment (initvector) [inline applications/vanilla/views/post/discussion.php]: "message" is a term normally associated with conversations.

File: applications/vanilla/views/post/comment.php [hunk 2]
```diff
@@ -28,7 +28,7 @@
                     echo $this->Form->errors();
                     $this->fireEvent('BeforeBodyField');
 
-                    echo $this->Form->bodyBox('Body', ['Table' => 'Comment', 'FileUpload' => true]);
+                    echo $this->Form->bodyBox('Body', ['Table' => 'Comment', 'FileUpload' => true, 'placeholder' => t('Type your comment'), 'title' => t('Type your comment')]);
```
Comment (initvector) [inline applications/vanilla/views/post/comment.php]: Vanilla typically ends sentences like this with a period.

File: applications/vanilla/views/post/discussion.php (hunk 1 above)
Comment (slafleche) [reply applications/vanilla/views/post/discussion.php]: I had checked with @tburry. We have multiple types of posts that all use this same string, so message seemed like a good neutral word for all of these post types.

File: applications/vanilla/views/post/comment.php (hunk 2 above)
Comment (slafleche) [reply applications/vanilla/views/post/comment.php]: @tburry do you want the period for the placeholders?

Comment (tburry) [reply applications/vanilla/views/post/comment.php]: I think placeholders are better without periods.

File: applications/vanilla/views/post/discussion.php (hunk 1 above)
Comment (tburry) [reply applications/vanilla/views/post/discussion.php]: Message is fine in this context.

**Pull request merged**